helm/cloud-provider-vsphere/values.yaml

```diff
@@ -10,3 +10,11 @@ global:
     zone: ""
     # openssl s_client -connect <vcenter>:443 < /dev/null 2>/dev/null | openssl x509 -fingerprint -sha1 -noout
     thumbprint: ""
+
+kube-vip:
+  enabled: true
+
+kube-vip-cloud-provider:
```

Comment:
Shouldn't it be `kubeVipCloudProvider`?

Reply:
This has to refer to the name of sub chart and it is how charts are named. I checked other multi-chart apps and saw this convention.